tests/common/plugins/sanity_check/checks.py

```diff
@@ -278,6 +285,166 @@ def _check_monit_services_status(check_result, monit_services_status):
+                                                        ip_snd=upper_tor_mgmt_ip,
+                                                        ip_tgt=upper_tor_ping_tgt_ip,
+                                                        hw_snd=upper_tor_intf_mac)
+        lower_tor_exp_pkt = testutils.simple_arp_packet(eth_dst='ff:ff:ff:ff:ff:ff',
+                                                        eth_src=lower_tor_intf_mac,
+                                                        ip_snd=lower_tor_mgmt_ip,
+                                                        ip_tgt=lower_tor_ping_tgt_ip,
+                                                        hw_snd=lower_tor_intf_mac)
+        
+        ptf_arp_pkt = testutils.simple_arp_packet(ip_tgt=ptf_arp_tgt_ip,
+                                                ip_snd=ptf_arp_tgt_ip,
+                                                arp_op=2)
+
+        # Run tests with upper ToR active
+        toggle_simulator_port_to_upper_tor(mux_server_url, mux_simulator_index)
+
+        try:
+            pytest_assert(get_simulator_read_side(mux_server_url, mux_simulator_index) == 1)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Unable to switch active link to upper ToR'
+            return results
+
+        # Ping from both ToRs, expect both messages to reach PTF
+        upper_tor_host.shell(ping_cmd.format(tor_mux_intf, upper_tor_ping_tgt_ip))
+        try:
+            testutils.verify_packet(ptfadapter, upper_tor_exp_pkt, ptf_port_index)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Packet from active upper ToR not received'
+            return results
+
+        lower_tor_host.shell(ping_cmd.format(tor_mux_intf, lower_tor_ping_tgt_ip))
+        try:
+            testutils.verify_no_packet(ptfadapter, lower_tor_exp_pkt, ptf_port_index)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Packet from standby lower ToR received'
+            return results
+
+        # Send dummy ARP packets from PTF to ToR. Ensure that ARP is learned on active ToR only
+        upper_tor_host.shell("ip neigh flush all")
+        lower_tor_host.shell("ip neigh flush all")
+        testutils.send_packet(ptfadapter, ptf_port_index, ptf_arp_pkt)
+
+        upper_tor_arp_table = upper_tor_host.switch_arptable()['ansible_facts']['arptable']['v4']
+        lower_tor_arp_table = lower_tor_host.switch_arptable()['ansible_facts']['arptable']['v4']
+        try:
+            pytest_assert(ptf_arp_tgt_ip in upper_tor_arp_table)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Packet from PTF not received on active upper ToR'
+            return results
+
+        try:
+            pytest_assert(ptf_arp_tgt_ip in lower_tor_arp_table)
```

Comment:
`ptf_arp_tgt_ip in lower_tor_arp_table` --> `ptf_arp_tgt_ip not in lower_tor_arp_table`?

Reply:
See https://github.com/Azure/sonic-mgmt/pull/2862#discussion_r579359928